adding clarification over xmlns:connection="true" (#50)
mulesoft/docs-mule-sdk: modules/ROOT/pages/xml-sdk.adoc

</module>
----

> Bear in mind that marking multiple global elements with `xmlns:connection="true"` will make the compilation fail, as there must be only one.

Reviewer: ```suggestion
[NOTE]
Marking multiple global elements with `xmlns:connection="true"` makes the compilation fail, as there can be only one.
```
